CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## [2.0.120230418]
+- [Enable auto review by chatgpt](https://github.com/Azure/Bridge-To-Kubernetes/pull/232)
```

Comment:
☕️ thank you for this, and I might have completely missed this last time but all these changes are in B2K repo, so we can refer the release email with a link saying, this **vscode** release is simply the release of B2K new cli. 

Thanks heaps, let’s talk soon as well.

Reply:
agreed, I thought the same thing. I will update them.

Reply:
❤️☕️ Thank you for the quick loop back, done.